locales/cs_CZ/LC_MESSAGES/messages.po

```diff
@@ -18,7 +18,7 @@ msgstr ""
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Language: cs\n"
-"Plural-Forms: nplurals=3; plural=(n==1) ? 0 : (n>=2 && n<=4) ? 1 : 2;\n"
+"Plural-Forms: nplurals=4; plural=(n == 1 && n % 1 == 0) ? 0 : (n >= 2 && n <= 4 && n % 1 == 0) ? 1: (n % 1 != 0 ) ? 2 : 3;\n"
```

Comment:
This `Plural-Forms` string was exported by Transifex, I did not check it in detail. Transifex changed the plural forms of some slavic languages some month ago and added one more plural form to conform to [Unicode CLDR](http://www.unicode.org/cldr/charts/latest/supplemental/language_plural_rules.html).

The new rules seem to be equal to the [rules for Czech](http://www.unicode.org/cldr/charts/latest/supplemental/language_plural_rules.html#cs).

The check `n % 1 == 0` seems to be necessary because decimal numbers have an own plural form. I think that's what they express with `v = 0` in the CLDR.

Reply:
Just that there is no v in the new formula. The old formula is the one that is still used by KDE, so I'm very sceptic about that change.

Reply:
@dbrvnistan Can you help us with this question? Some sources say there are [3 plural forms](http://docs.translatehouse.org/projects/localization-guide/en/latest/l10n/pluralforms.html) for Czech (singular + 2 plural), other sources say there are [4 plural forms](http://www.unicode.org/cldr/charts/latest/supplemental/language_plural_rules.html#cs) (singular + 2 plural + 1 for non-integer values). Which one is correct?

Reply:
I guess https://lingonborough.com/how-polish-plurals-in-mate-went-broken/ is about the same change?

Reply:
Yes, looks like an explanation of what happened.

Reply:
I removed the plural form updates.